app/helpers/automated_tests_helper.rb

```diff
@@ -48,12 +48,11 @@ def update_test_groups_from_specs(assignment, test_specs)
         display_output = extra_data_specs['display_output'] || TestGroup.display_outputs.keys.first
         test_group_name = extra_data_specs['name'] || TestGroup.model_name.human
         criterion_id = nil
-        criterion_type = nil
         if !extra_data_specs['criterion'].nil? && extra_data_specs['criterion'].include?('_')
```

Comment:
Remove the second clause. Just set `criterion_id = extra_data_specs['criterion']` (this goes with my previous comment).

Reply:
What do you mean by this goes with your previous comment?